salt/cloud/clouds/azurearm.py

```diff
@@ -117,9 +118,12 @@
     from azure.mgmt.storage import StorageManagementClient
     from azure.mgmt.web import WebSiteManagementClient
     from msrestazure.azure_exceptions import CloudError
-    from azure.multiapi.storage.v2016_05_31 import CloudStorageAccount
-    from azure.cli import core
-    HAS_LIBS = LooseVersion(core.__version__) >= LooseVersion("2.0.12")
+    if pkgutil.find_loader('azure.multiapi'):
```

Comment:
What about using the `salt.utils.path.which` function instead here?

Reply:
Hi @rallytime , if I'm not mistaken, `salt.utils.path.which` looks for an executable, whereas this code looks for an importable module. I don't see how `which` would be useful here, or am I missing something?